journal-writings/original/19-epistle.adoc

```diff
@@ -477,17 +475,17 @@ signified "`that the blood of all the prophets
 which was shed from the foundation of the world,
 should be required of that generation, from the blood of Abel,
 unto the blood of Zacharias, which perished between the altar and the temple.`"
-
 Tender compassion fills my heart toward my fellow-creatures estranged
 from the harmonious government of the Prince of peace,
 and a labour attends me, that they may be gathered to this peaceable habitation.
 
 In being inwardly prepared to suffer adversity for Christ`'s sake,
-and weaned from a dependance on the arm of flesh,
+and weaned from a dependence on the arm of flesh,
 we feel that there is a rest for the people of God,
 and that it stands in a perfect resignation of ourselves to his holy will.
 In this condition all our wants and desires are bounded by pure wisdom,
 and our minds are wholly attentive to the counsel of Christ inwardly communicated.
+
```

Comment:
Let's delete this break.

Reply:
oh for heaven's sake! I'm all sweaty over here! (not really)... but my lines/numbers don't appear to to match what you're saying to do...? anyway, I changed things and sorry to ask you to have to go through them all again but I am not confident that I changed what you asked me to